components/tidb_query_datatype/src/codec/data_type/mod.rs

```diff
@@ -193,7 +193,7 @@ pub trait EvaluableRet: Clone + std::fmt::Debug + Send + Sync + 'static {
     type ChunkedType: ChunkedVec<Self>;
     /// Converts a vector of this concrete type into a `VectorValue` in the same type;
     /// panics if the varient mismatches.
-    fn into_vector_value(vec: Self::ChunkedType) -> VectorValue;
+    fn cast_into_vector_value(vec: Self::ChunkedType) -> VectorValue;
```

Comment:
The new name is still ambiguous. What about `cast_chunk_to_vector_value`?

Reply:
Sounds good